plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationIndicatorLayerRenderer.kt

```diff
@@ -25,6 +25,7 @@ internal class LocationIndicatorLayerRenderer(
   override fun initializeComponents(style: StyleInterface) {
     this.style = style
     setupBitmaps()
+    setOpacity(puckOptions.opacity.toDouble())
```

Comment:
I'm not convinced this is the right place. Suggestions welcomed.

Reply:
I'd put in inside `setupBitmaps()` near 
```kotlin
layer.topImage(TOP_ICON)
layer.bearingImage(BEARING_ICON)
layer.shadowImage(SHADOW_ICON)
```